data/faq/writing_a_great_proposal.md

```diff
@@ -14,3 +14,23 @@ We want this to be a fun and educating experience for you!
 
 Use the"draft" feature on the GSoC system, submit your draft early, get a
 lot of helpful feedback from us!
+
+
+#### Writing the proposal
+
+Please follow the **proposal template** at [PSF-ApplicationTemplate2016](https://wiki.python.org/moin/SummerOfCode/ApplicationTemplate2016). All fields are required if not explicitly indicated otherwise. While following the application template you can consider doing an illustration of your proposed solution as it makes the application much more intuitive.
```

Comment:
We want to give this in the org profile at google right? But we can't link there yet if I understand correctly